app/controllers/concerns/phone_confirmation_flow.rb

```diff
@@ -76,7 +76,7 @@ def send_confirmation_code
     job.perform_later(
       code: confirmation_code,
       phone: unconfirmed_phone,
-      otp_created_at: current_user.direct_otp_sent_at.to_s
+      otp_created_at: (current_user.direct_otp_sent_at || Time.zone.now).to_s
```

Comment:
Yeah I mean there could be a small difference. We don't know for sure that this job is enqueued right when the OTP is generated. I can imagine a future where we do a bunch of stuff in the controller after the OTP is generated and then enqueue the job. It would be surprising if that took much time, but it is technically possible.

In short: using that timestamp is how our system determines whether or not an OTP is still valid, so it is the most accurate way to know if an OTP is valid. But assuming the timestamp and the current time are pretty close is not a bad assumption to make.